guides/deploying-to-digital-ocean-with-ansible.md

```diff
@@ -0,0 +1,167 @@
+
+Copy the newly created IP address and verify that you can log in into the new server by executing:
+
+```
+ssh root@XXX.XXX.XXX.XXX
+```
+
+## Prepare the Control Node
+
+Ansible requires a control node, which is a computer that manages a remote host. This guide assumes a Mac laptop/desktop as the control node.
+
+Install Ansible using [homebrew](https://brew.sh), this guide assumes some familiarity with Ansible, if you need an introduction to basic concepts click [here](https://www.ansibletutorials.com).
+
+`brew install ansible`
+
+Also install python3 to avoid deprecation warnings.
+
+`brew install python3`
+
+## Configure the remote host to be managed with Ansible
+
+On the control node (i.e. a developer's machine) create an inventory file in which `python3` is specified as the interpreter. On your machine, create a new file named `hosts` at `/etc/ansible` that shall act as your ansible inventory file.
+```
+touch /etc/ansible/hosts
+```
+
+Add the following content to the inventory file:
+```
+[servers]
+reaction.server
+
+[servers:vars]
+ansible_python_interpreter=/usr/bin/python3
+[web]
+```
+
+Edit your hosts file (your local DNS hosts file, NOT your newly created Ansible hosts/inventory file)
+```
+sudo vim /etc/hosts
+```
+
+And add an entry for the DigitalOcean droplet, ensuring to substitute the XXX octet's for your droplets external IP 
+address.
+
+```
+XXX.XXX.XXX.XXX reaction.server
+```
+
+Verify that Ansible can communicate with your remote host by executing:
+
+```
+ansible all -m ping -u root
+```
+
+Your should see output similar to:
+
+```
+reaction.dev | SUCCESS => {
+    "changed": false,
+    "ping": "pong"
+}
+```
+
+## Set Ansible Environment Variables
+
+Before executing the Ansible playbook, it's necessary to set variables that are specific to your deployment. Find the
+`vars` section in the `reaction.yml` playbook and update as necessary, below is a list of the variable
+that need to be updated and a description of each.
+
+| Variable               | Description                                                                 |
+| ---------------------- | ----------------------------------------------------------------------------|
+| do_auth_token          | The Authentication token for the Digital Ocean API                          |
+| email                  | An email address to receive SSL certificate notifications                   |
+| domain                 | Your registered domain                                                      |
+
+For the rest of the variables, the default values should be used, DO NOT change otherwise the playbook might fail.
+
+## Execute the playbook
+
+Now it's time to execute the `reaction.yml` playbook, which automates most of the tedious server configuration tasks, execute the following command:
```

Comment:
Same suggestion re: 2 sentences.

Now it's time to execute the `reaction.yml` playbook, which automates most of the tedious server configuration tasks. Execute the following command: